app/components/Header/Header.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Logo from "../Logo/Logo";
+import classNames from "classnames";
+import { HeaderType } from "./Header.types";
+import NavbarMenu from "../NavbarMenu/NavbarMenu";
+
+const Header = ({ logo, links, ActionIcon, extraClasses }: HeaderType) => {
+  let HeaderClass = classNames("text-2xl text-lightBlue");
+  return (
+    <header className="flex justify-evenly  bg-darkBlue py-4">
+      <div className="flex gap-2">
+        <Logo logo={logo} />
```

Comment:
Logo has a logo prop, it doesn't sound right.
You can name the as props or something